students/idcrickmore/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,16 @@
+def fizzbuzz(x):
+    if x%3==0 and x%5==0:
```

Comment:
(pep8 style guidelines) should have spaces: `if x % 3 == 0`

there are online linters and also built-in for some IDEs (sublime, pycharm, atom) that can check for pep8 style violations.